solr/core/src/java/org/apache/solr/core/SolrCore.java

```diff
@@ -491,22 +490,27 @@ public long getIndexSize() {
   }
 
   public long getCachedIndexSize() {
```

Comment:
I think this method should be named simply getIndexSize(), and then the method that is currently named that could be computeIndexSize (plus a javadoc to ask the caller to call getIndexSize?  WDYT?  Thus the cached aspect is an implementation detail.